proposals/0245-sharing-wifi-ssid-and-password.md

```diff
@@ -167,244 +163,19 @@ If the result of the intersection of these parameters is a single value, which i
-    <td>External</td>
-  </tr>
-  <tr>
-    <td rowspan="2">false</td>
-    <td rowspan="2">false</td>
-    <td>Vehicle/Mobile/omitted</td>
-    <td>Vehicle/Mobile/omitted</td>
-    <td>not be initialized</td>
-  </tr>
-  <tr>
-    <td>External</td>
-    <td>Vehicle/Mobile/omitted</td>
-    <td>External</td>
-  </tr>
-  <tr>
-    <td rowspan="4">false</td>
-    <td rowspan="4">false</td>
-    <td>true</td>
-    <td>true</td>
-    <td rowspan="4">any</td>
-    <td rowspan="4">any</td>
-    <td rowspan="4">not be initialized</td>
-  </tr>
-  <tr>
-    <td>true</td>
-    <td>false</td>
-  </tr>
-  <tr>
-    <td>false</td>
-    <td>true</td>
-  </tr>
-  <tr>
-    <td>false</td>
-    <td>false</td>
-  </tr>
-</table>
-
-
-__Note:__ `preferredNetworkHost` parameter will be considered only if `hostingWiFiSupported` and `autoJoinWiFiSupported` is `true` in vehicle and mobile application capabilities.
-
-In any case, from the above table, SDL Core has to check whether the network host is allowed in the ini file.
-
-If both HMI and mobile application can host the network and both network host values are allowed in the ini file, SDL Core checks `preferredNetworkHost` from capabilities. If this parameter coincides for mobile and HMI capabilities, then this value has to be taken as a network host.
-
-If the `preferredNetworkHost` is omitted in mobile application network capabilities, vehicle preference will be chosen.
-
-If the `preferredNetworkHost` is omitted in vehicle network capabilities, mobile application preference will be chosen.
-
-If for both vehicle and mobile application network capabilities `preferredNetworkHost` parameter is omitted, a decision will be based on ini file configuration.
-
-If `preferredNetworkHost` of mobile application network capabilities contradict the corresponding parameter of vehicle network capabilities, a decision will be based on ini file configuration.
+#### Determining which device should host the network
+
+Once the head unit receives the RegisterAppInterface and the NetworkingCapabilities it must decide who should host. Each bullet point assumes that the network configurations can be supported by both devices (frequencies, data fallback, etc). These bullet points are just a suggestion, an OEM can determine the situation themselves.
+
+* MOBILE (assuming canHostWiFiNetwork in NetworkingAbilities is set to true from mobile)
+  - Core set MOBILE as the only device in the INI file
+  - If Core set BOTH and the mobile device does NOT support dataFallbackSupported
+  - If Core set BOTH and the mobile device set dataUsagePreference to MOBILE value
+* VEHICLE (assuming canJoinNetwork in NetworkingAbilities is set to true from mobile)
+  - Core set VEHICLE as the only device in the INI file
+  - If Core set BOTH and the mobile device DOES support dataFallbackSupported
+  - If Core set BOTH and the mobile device sets dataUsagePreference as VEHICLE
+  - If any other settings mismatch, this will be the default

```

Comment:
Please update to include backticks as used in [this comment](https://github.com/smartdevicelink/sdl_evolution/issues/799#issuecomment-588434670).

Reply:
Added Additional Information.
Also added Android Permission Issue to Troubleshuting section

Reply:
Backticks were not added to lines 172 and 173.  Please add.

Reply:
@LuxoftAKutsan backticks still need to be added to lines 172 - 177.

Reply:
@LuxoftAKutsan this comment still needs to be addressed, thanks!

Reply:
@theresalech added 3342202